lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -521,6 +521,45 @@ describe('BasicLtiLaunchApp', () => {
         // Dialog prompt should remain.
         await dialogShouldRemain(wrapper);
       });
+
+      it('disables "Authorize" button while re-fetching content and groups', async () => {
```

Comment:
the tests for this file are getting crazy!

Reply:
> the tests for this file are getting crazy!

Indeed. I think it may be worth looking at splitting some functionality out of this component soon or otherwise refactoring how it works.